src/docker-in-docker/devcontainer-feature.json

```diff
@@ -50,6 +45,16 @@
             "type": "boolean",
             "default": true,
             "description": "Install Docker Buildx"
+        },
+        "mobyBuildxVersion": {
+            "type": "string",
+            "default": "0.12.0",
```

Comment:
Are we planning to reset it to `latest` once the upstream issue is fixed, or are we going to deterministically pick and maintain the default here?

Reply:
I believe @samruddhikhandaSam has #839 set up to change this to latest as soon as the package manifest is updated

Reply:
Thanks.
Realized that this PR is for adding `composePackageVersion`